docs/content/preview/explore/query-1-performance/pg-stat-statements.md

```diff
@@ -67,32 +67,33 @@ yugabyte=# \d pg_stat_statements;
- max_time            | double precision |           |          |
- mean_time           | double precision |           |          |
- stddev_time         | double precision |           |          |
- rows                | bigint           |           |          |
- shared_blks_hit     | bigint           |           |          |
- shared_blks_read    | bigint           |           |          |
- shared_blks_dirtied | bigint           |           |          |
- shared_blks_written | bigint           |           |          |
- local_blks_hit      | bigint           |           |          |
- local_blks_read     | bigint           |           |          |
- local_blks_dirtied  | bigint           |           |          |
- local_blks_written  | bigint           |           |          |
- temp_blks_read      | bigint           |           |          |
- temp_blks_written   | bigint           |           |          |
- blk_read_time       | double precision |           |          |
- blk_write_time      | double precision |           |          |
+                     View "public.pg_stat_statements"
+       Column         |       Type       | Collation | Nullable | Default
+----------------------+------------------+-----------+----------+---------
+ userid               | oid              |           |          |
+ dbid                 | oid              |           |          |
+ queryid              | bigint           |           |          |
+ query                | text             |           |          |
+ calls                | bigint           |           |          |
+ total_time           | double precision |           |          |
+ min_time             | double precision |           |          |
+ max_time             | double precision |           |          |
+ mean_time            | double precision |           |          |
+ stddev_time          | double precision |           |          |
+ rows                 | bigint           |           |          |
+ shared_blks_hit      | bigint           |           |          |
+ shared_blks_read     | bigint           |           |          |
+ shared_blks_dirtied  | bigint           |           |          |
+ shared_blks_written  | bigint           |           |          |
+ local_blks_hit       | bigint           |           |          |
+ local_blks_read      | bigint           |           |          |
+ local_blks_dirtied   | bigint           |           |          |
+ local_blks_written   | bigint           |           |          |
+ temp_blks_read       | bigint           |           |          |
+ temp_blks_written    | bigint           |           |          |
+ blk_read_time        | double precision |           |          |
+ blk_write_time       | double precision |           |          |
+ yb_latency_histogram | jsonb            |           |          |
  ```
 
 Describe the view's columns, including storage type and description:
```

Comment:
I missed this one, please update this section to include the new column as well. 
I recorded in the https://docs.google.com/document/d/19ozvZ6F0kqzGPLl8CGiioQs60Q7qX0HtQpdKYleeHp4/edit#heading=h.2gahxj34y6i8

Reply:
On second thought: We should remove "Describe the view's columns, including storage type and description:" 
It does not add any value. 

Alternatively we should add the column description similar to https://www.postgresql.org/docs/11/pgstatstatements.html#id-1.11.7.38.6 .